README.md

```diff
@@ -716,6 +807,26 @@ add_filter( 'vip_block_data_api__rest_permission_callback', function( $is_permit
 
 ---
 
+### `vip_block_data_api__content_filter_block`
+
+Filter out blocks from the output of the Block Data API.
+
+```php
+/**
+ * Filter out certain blocks from the blocks output
+ *
+ * @param array  $is_block_included A boolean value where true means the block is included, and false to filter it out.
+ * @param string $block_name    The name of the parsed block, e.g. 'core/paragraph'.
+ * @param string $block         The result of parse_blocks() for this block.
+ *                              Contains 'blockName', 'attrs', 'innerHTML', and 'innerBlocks' keys.
+ */
+apply_filters( 'vip_block_data_api__content_filter_block', $is_block_included, $block_name, $block );
+```
+
+This is useful when it's necessary to filter out the blocks given back by the Block Data API, including in the inner blocks of a block as well.
```

Comment:
Can we add a couple of examples here for implementing a basic allow and blocklist via this filter?